spec/support/vacancy_helpers.rb

```diff
@@ -229,6 +239,7 @@ def vacancy_json_ld(vacancy)
       "@type": "JobPosting",
       title: vacancy.job_title,
       jobBenefits: vacancy.benefits,
+      jobBenefitsDetails: vacancy.benefits_details,
```

Comment:
I think this would have to remain as `jobBenefits`.

Take a look at https://schema.org/JobPosting